### ISSUE TITLE
Forbid "console" funcs

### ISSUE THREAD

**Issue by dpordomingo** [src-d/gitbase-playground#28]
from https://github.com/src-d/gitbase-playground/pull/16#discussion_r187585501

It could be a good idea 
- [ ] to forbid `console` commands (`.eslintrc.js::rules['no-console']`), and
- [ ] to provide [a logger wrapper as in  `CAT`](https://github.com/src-d/code-annotation/blob/master/src/services/log.js)?

**Comment by dpordomingo** [comment]
but ci does not fail with warnings, does it?

**Comment by smacker** [comment]
nope. And this allows PR like this: https://github.com/src-d/gitbase-playground/pull/35/files#diff-48bb2e3e3e5708d8083b1cff1fff3949R81 which isn't bad at this stage of the project. :)

**Comment by dpordomingo** [comment]
The point of this issue is to address this matter whenever we're ready to do so
